test/batches_SUITE.erl

```diff
@@ -43,8 +43,8 @@ groups() ->
     ].
 
 init_per_suite(Config) ->
-    application:ensure_all_started(telemetry),
     meck:new(mongoose_metrics, [stub_all, no_link]),
+    meck:new(mongoose_instrument, [stub_all, no_link]),
```

Comment:
1. For consistency with other small tests, and to check that the instrumentation is not called with wrong arguments, I'd rather just start `mongoose_instrument` here without mocking it.
2. Maybe we don't need `mongoose_metrics` now? Did you check that?

Reply:
1. That's a good idea. Thanks for the tip!
2. The `mongoose_metrics` are still needed because the async pools are still using them.